src/resources/postcss/tickets-attendees-print.pcss

```diff
@@ -51,11 +49,35 @@
 	}
 
 	#tribe-attendees-summary .welcome-panel-column {
-		width: 100%;
 		padding: 0;
 		float: none;
 	}
 
+	#tribe-attendees-summary {
+		margin-bottom: 30px;
+		.welcome-panel-column {
```

Comment:
Add a line break between the properties of the element itself and the start of any nested selectors?